ext/ddtrace.stub.php

```diff
@@ -23,6 +23,23 @@
      */
     const DBM_PROPAGATION_FULL = UNKNOWN;
 
+    class SpanEvent implements \JsonSerializable {
```

Comment:
We don't need Json serialization I think - that was as far as I'm aware an implementation artifact of span links.